op-batcher/batcher/espresso.go

```diff
@@ -192,6 +194,49 @@ func (s *espressoTransactionSubmitter) SubmitTransaction(job *espressoCommon.Tra
 	}
 }
 
+// Evaluation result for a job.
+type JobEvaluation int
+
+const (
+	// Continue handling the current job.
+	Handle JobEvaluation = iota
+	// Retry the submission.
+	RetrySubmission
+	// Retry the verification.
+	RetryVerification
+	// Skip the current job and proceed to the next one.
+	Skip
+)
+
+// Evaluate the submission job.
+func evaluateSubmission(jobResp espressoSubmitTransactionJobResponse) JobEvaluation {
+	err := jobResp.err
+
+	// If there's no error, continue handling the submission.
+	if err == nil {
+		return Handle
+	}
+
+	msg := err.Error()
+
+	// If the transaction is invalid, skip the submission.
+	if strings.Contains(msg, "json:") {
+		log.Warn("json.Marshal fails, skipping", "msg", msg)
+		return Skip
+	}
+
+	// If the request is invalid, skip the submission.
```

Comment:
Does this only happen if we have a bug in our code or if there is some API change for the Espresso QS node?

Reply:
Likely the latter case, but the logged error message would provide more info. I just updated the comment!